app/server/models/user.py

```diff
@@ -0,0 +1,466 @@
+from server import db, sentry, celery_app
+from typing import Union
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.dialects.postgresql import JSON, JSONB
+from itsdangerous import TimedJSONWebSignatureSerializer, BadSignature, SignatureExpired
+import pyotp
+from flask import current_app
+import datetime, bcrypt, jwt, random, string
+
+from server.utils.misc import encrypt_string, decrypt_string
+from server.utils.access_control import AccessControl
+from server.utils.phone import proccess_phone_number
+from server.models.utils import ModelBase, ManyOrgBase, user_transfer_account_association_table
+from server.models.models import Organisation, BlacklistToken
+from server.exceptions import (
+    RoleNotFoundException,
+    TierNotFoundException,
+    NoTransferCardError
+)
+from server.constants import (
+    ACCESS_ROLES
+)
+
+# TODO(refactor): circular dependency when in util.user
+def get_transfer_card(public_serial_number):
```

Comment:
don't know if this is correct fix. this was in util.user originally, but then user was importing util.user which was importing user...

Reply:
This is the best overview i've ever seen on this issue:
https://stackoverflow.com/questions/22187279/python-circular-importing

I've used that process in a few places, for example in /utils/credit_transfers.py

`    transfer = models.CreditTransfer(amount=transfer_amount,
                                     token=token,
                                     sender_transfer_account=outbound_transfer_account,
                                     recipient_user=receive_account,
                                     transfer_type=models.TransferTypeEnum.DISBURSEMENT, uuid=uuid)`

That being said, i do wonder if minimising the use of helper function files now that we have a few seperate models files may actually help this in a far easier manner.

Reply:
got it, so should we start using `import x` instead of `from x import y`?